### PR TITLE
use a different image in previews

### PR DESCRIPTION
**Pull request by foldleft**
fixes: https://github.com/vector-im/riot-web/issues/12014

replaces: https://github.com/vector-im/riot-web/pull/13461

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const webpack = require("webpack");
 
 let og_image_url = process.env.RIOT_OG_IMAGE_URL;
-if (!og_image_url) og_image_url = 'https://riot.im/app/themes/riot/img/logos/riot-im-logo-black-text.png';
+if (!og_image_url) og_image_url = 'https://about.riot.im/images/opengraph.png';
```
Comment (jryans) [inline webpack.config.js:9]: Perhaps it's better to keep the app self-contained by copying this asset into the logos folder? If we redesign about.riot.im, some day, we're likely to forget we've depended on this path.

**Comment by ara4n** [comment]
worth noting https://riot.im/app/themes/riot/img/logos/opengraph.png is 404ing

**Comment by jryans** [comment]
> worth noting https://riot.im/app/themes/riot/img/logos/opengraph.png is 404ing

Right, it normally wouldn't have appeared until release. I went ahead and manually copied it from develop to app, so should be good now.

**Comment by ara4n** [comment]
thanks

**Pull request merged**